content/en/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md

```diff
@@ -22,43 +22,19 @@ vacated by the evicted critical add-on pod or the amount of resources available
 
 {{% capture body %}}
 
-## Rescheduler: guaranteed scheduling of critical add-ons
-**Rescheduler is deprecated as of Kubernetes 1.10 and will be removed in version 1.12 in
+### Marking pod as critical using critical pod annotation. 
+
+**Critical pod annotation is deprecated as of Kubernetes 1.13 and will be removed in version 1.14 in
 accordance with the [deprecation policy](/docs/reference/deprecation-policy) for beta features.**
 
 **To avoid eviction of critical pods, you must
 [enable priorities in scheduler](/docs/concepts/configuration/pod-priority-preemption/)
-before upgrading to Kubernetes 1.10 or higher.**
-
-Rescheduler ensures that critical pods created by DaemonSet controller are always scheduled
-(assuming the cluster has enough resources to run the critical add-on pods in the absence of regular pods).
-If the scheduler determines that no node has enough free resources to run the critical add-on pod
-given the pods that are already running in the cluster
-(indicated by critical add-on pod's pod condition PodScheduled set to false, the reason set to Unschedulable)
-the rescheduler tries to free up space for the DaemonSet critical pod by evicting some pods; then the scheduler will schedule the add-on pod.
-
-To avoid situation when another pod is scheduled into the space prepared for the critical add-on,
-the chosen node gets a temporary taint "CriticalAddonsOnly" before the eviction(s)
-(see [more details](https://git.k8s.io/community/contributors/design-proposals/scheduling/taint-toleration-dedicated.md)).
-Each critical add-on has to tolerate it,
-while the other pods shouldn't tolerate the taint. The taint is removed once the add-on is successfully scheduled.
-
-*Warning:* currently there is no guarantee which node is chosen and which pods are being killed
-in order to schedule critical pods, so if rescheduler is enabled your pods might be occasionally
-killed for this purpose. Please ensure that rescheduler is not enabled along with priorities & preemptions in default-scheduler as rescheduler is oblivious to priorities and it may evict high priority pods, instead of low priority ones.
+before upgrading to Kubernetes 1.11 or higher.**
 
-## Config
-
-Rescheduler doesn't have any user facing configuration (component config) or API.
-
-### Marking pod as critical when using Rescheduler. 
 
 To be considered critical, the pod has to run in the `kube-system` namespace (configurable via flag) and
 
-* have the `scheduler.alpha.kubernetes.io/critical-pod` annotation set to empty string, and
-* have the PodSpec's `tolerations` field set to `[{"key":"CriticalAddonsOnly", "operator":"Exists"}]`.
-
-The first one marks a pod a critical. The second one is required by Rescheduler algorithm.
+* have the `scheduler.alpha.kubernetes.io/critical-pod` annotation set to empty string.
```

Comment:
We should also change this and say that the pod must have its priorityClassName set to `system-node-critical` or `system-cluster-critical` and link to the pod priority doc.

Reply:
The subsequent section says talks about using priority.

Reply:
I see. I think we should remove this paragraph given that we no longer want users to use this annotation.

Reply:
Shouldn't we remove this when we are deleting the `critical pod annotation` completely.

Reply:
Given that these docs are versioned and we are changing this for 1.13, I think it makes sense to delete this paragraph. We don't want people to use this deprecated feature in 1.13.

Reply:
Ok, done. Thanks